src/pages/organize/[orgId]/campaigns/[campId]/surveys/[surveyId]/index.tsx

```diff
@@ -27,9 +34,59 @@ interface SurveyPageProps {
   surveyId: string;
 }
 
-const SurveyPage: PageWithLayout<SurveyPageProps> = () => {
-  return <></>;
+const SurveyPage: PageWithLayout<SurveyPageProps> = ({
+  campId,
+  orgId,
+  surveyId,
+}) => {
+  const model = useModel(
+    (env) => new SurveyDataModel(env, parseInt(orgId), parseInt(surveyId))
+  );
+  const onServer = useServerSide();
+
+  if (onServer) {
+    return null;
+  }
+
+  const { data: survey } = model.getData();
+
+  if (!survey) {
+    return null;
+  }
+
+  return (
+    <Box
+      alignItems="center"
+      display="flex"
+      justifyContent="center"
+      paddingTop={8}
+    >
+      {survey.elements.length ? (
+        <></>
+      ) : (
+        <Box alignItems="center" display="flex" flexDirection="column">
+          <QuizOutlined
+            color="secondary"
+            sx={{ fontSize: '8em', paddingBottom: 2 }}
+          />
+          <Typography color="secondary">
+            <Msg id="pages.organizeSurvey.overview.noQuestions.title" />
+          </Typography>
+          <Link
+            href={`/organize/${orgId}/campaigns/${campId}/surveys/${surveyId}/questions`}
+            sx={{ marginTop: 4 }}
+            underline="none"
+          >
+            <Button variant="contained">
+              <Msg id="pages.organizeSurvey.overview.noQuestions.button" />
+            </Button>
+          </Link>
+        </Box>
+      )}
+    </Box>
```

Comment:
I think this should be moved to it's own component, to keep the page component short and avoid avoid complex ternaries.